internal/cmd/commands/authmethodscmd/oidc_funcs.go

```diff
@@ -129,6 +132,12 @@ func extraOidcFlagsFuncImpl(c *OidcCommand, set *base.FlagSets, _ *base.FlagSet)
 				Target: &c.flagClaimsScopes,
 				Usage:  `The optional claims scope requested. May be specified multiple times.`,
 			})
+		case accountClaimMaps:
+			f.StringSliceVar(&base.StringSliceVar{
+				Name:   accountClaimMaps,
+				Target: &c.flagAccountClaimMaps,
+				Usage:  `The optional account claim maps from custom claims to the standard claims of sub, name and email.  These maps are represented as key=value where the key equals the from-claim and the value equals the to-claim.  For example "oid=sub". May be specified multiple times.`,
```

Comment:
```suggestion
				Usage:  `The optional account claim maps from custom claims to the standard claims of sub, name and email.  These maps are represented as key=value where the key equals the from-claim and the value equals the to-claim.  For example "oid=sub". May be specified multiple times for different to-claims.`,
```

Reply:
I'm not sure what good wording would be but consider calling out that the `from-claim` is the claim from the provider and the `to-claim` is what boundary treats the claim as.

Reply:
ty!